src/2014.ipynb

```diff
@@ -114,61 +127,49 @@
+      "174\n"
      ]
     }
    ],
    "source": [
-    "print(projects_strings[0:3])"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 10,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "174"
-      ]
-     },
-     "execution_count": 10,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "len(projects_strings)"
+    "print(projects_strings[0:3])\n",
+    "print(len(projects_strings))"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 68,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Reference:\n",
```

Comment:
Maybe this will help. You can also check out https://docs.python.org/3/library/re.html and https://pythex.org/